docs/content/latest/explore/ysql-language-features/gin.md

```diff
@@ -47,14 +47,12 @@ However, extension support is still in progress:
 
 ### CREATE INDEX
 
-Create the index using `USING ybgin` to specify the index access method:
+Create the index using `USING gin` to specify the index access method:
 
 ```sql
-CREATE INDEX ON mytable USING ybgin (mycol);
-```
+CREATE INDEX <optional name> ON <table> USING GIN (<column>);
```

Comment:
```suggestion
CREATE INDEX [name] ON <table> USING gin (<column>);
```

Please be consistent and say `USING gin`, not `USING GIN`.

Reply:
Is `CREATE INDEX <optional name> ON <table> USING GIN (<column>);` really preferable over the simpler `CREATE INDEX ON mytable USING gin (mycol);`?  Especially optional grammar items like index name are unnecessary to mention here in my opinion.

Reply:
Sure @jasonyb - i will take your suggestion and remove <optional>